apps/site/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "paths": { "@/*": ["./*"] },
+    "paths": { "@/*": ["./*"], "@generated/*": ["./.generated/*"] },
```

Comment:
Is this correct?

Reply:
outdated, I've removed all the generated files hacks 🚀